bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/wrappers/veneer/TestBigtableHBaseVeneerSettings.java

```diff
@@ -285,15 +285,12 @@ public void testWithNullCredentials() throws IOException {
     configuration.setBoolean(BIGTABLE_NULL_CREDENTIAL_ENABLE_KEY, true);
 
     BigtableDataSettings dataSettings =
-        ((BigtableHBaseVeneerSettings) BigtableHBaseVeneerSettings.create(configuration))
-            .getDataSettings();
-    assertTrue(
```

Comment:
was it intentional to delete this assertion?

Reply:
Credential will be an instance of FixedCredentialProvider so the test will fail. The other line checks for  .getCredentials() which should be null and basically means that we are using a NoCredentialsProvider

Reply:
thanks!